src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -62,10 +62,7 @@
   column-gap: var(--grid-column-gap);
   display: grid;
   grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
-  grid-template-rows: repeat(2, auto);
-  padding-bottom: 100px;
-  padding-top: 100px;
-  row-gap: 50px;
+  padding: 100px 0 98px;
```

Comment:
How about we leave the bottom padding as 100px? Even if the design shows 98px, I think that may just be a small inconsistency, and 100px would be almost visually identically yet more consistent in the code.